yarn.lock

```diff
@@ -7204,6 +7181,14 @@
     "@types/node" "*"
     form-data "^3.0.0"
 
+"@types/node-fetch@^2.6.0":
+  version "2.6.0"
+  resolved "https://registry.yarnpkg.com/@types/node-fetch/-/node-fetch-2.6.0.tgz#bf854a36a0d0d99436fd199e06612ef4e73591b6"
+  integrity sha512-HT+uU6V27wJFXgEqTk/+rVE1MWcp5bg7Yuz//43TZ2PjpQbQ8vDLwVmB+fSpgs83j/+p+rMIlDRo9TL3IexWMA==
+  dependencies:
+    "@types/node" "*"
+    form-data "^2.3.3"
```

Comment:
Nit: you could collapse `"@types/node-fetch@^2.5.7"` above and `"@types/node-fetch@^2.6.0"` here into a single dependency.

It looks like that would also implicitly get rid of `form-data@^3.0.0` too!

Reply:
Yep, thanks!